### ISSUE TITLE
Reproducible Result Problem

### ISSUE THREAD

**Issue by sriderya** [jfhealthcare/Chexpert#25]
Hi,
Thanks for the great work.

I trained the CheXpert dataset with this model by using pretrained model in the repo. I got the following results:
best1.ckpt -> 0.8554021060651713
best2.ckpt -> 0.8531909236770152
best3.ckpt -> 0.8546921678446425

Then, I want to train the same dataset with the same config and pretrained model. This time, the results were:
best1.ckpt -> 0.8405812891919311
best2.ckpt -> 0.8603955345253775
best3.ckpt -> 0.8383620949529762

Although seeds are set in the train.py for torch and I set the seed for numpy random, the results change every time I train the model. What could be the problem? How can I solve it? Thanks in advance.

```
torch.manual_seed(0)
torch.cuda.manual_seed_all(0)
np.random.seed(0)
```

**Comment by yil8** [comment]
@deadpoppy can you take a look?

**Comment by deadpoppy** [comment]
Hi, @sriderya Thank you for your question.
 
Do you use data augmentation? Our code does random image transformation, even though you are using the same data set, the data will still be changed. @sriderya

**Comment by sriderya** [comment]
I do not use data augmentation. I set use_transforms_type as None in the config. Nevertheless, I got different results as I mentioned in the question.

**Comment by deadpoppy** [comment]
I don't know the details of your experiment, so I'm sorry that it's difficult to give a solution. But I suggest you consider the following aspects:
1. Input data
2. Training strategy
2. Model saving strategy
@sriderya
